Dockerfile

```diff
@@ -19,16 +19,21 @@ RUN apt-get update && apt-get install -y \
     libclang-dev \
     clang
 
-RUN npm install -g typescript@3.9.7 \
+RUN yarn add --dev typescript@3.9.7 \
```

Comment:
No need to use NPM here. And no need for these to be global, either.
I'm not 100% certain if Typescript needs to be `--dev` or not? It worked locally, anyway.